python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -112,6 +112,11 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
     if missing_cred:
         raise Exception("Failed due to missing credentials %s" % str(missing_cred))
 
+    # Make sure that all the config options are really set.
+    # Some from plugins may not have taken during startup
```

Comment:
I believe (although the plugin loading is messy and multi-parted) that it happens in `bootstrap()`. The issue is likely coming from the fact that we set the config in a different way in these files to how we do in a real live Ganga session. We've also not until now been testing setting the config of plugins using this method until now. 

If this line is controversial, I'm happy to remove it as I don't believe it's needed for any of the tests in this PR.